### PR TITLE
add cedilla

### PR DESCRIPTION
**Pull request by kevinbarabash**
Addresses part of #2.

### CODE REVIEW

**Comment by edemaine** [comment]
Hmm, one question: it looks like this was added to 9 tables, but 8 fonts changed.  Explanation for the discrepancy?

File: src/fonts/OTF/TeX/makeFF [hunk 1]
```diff
@@ -593,6 +594,7 @@ $map{cmti10} = {
     6 => 0x3A3,             # \Sigma
     [7,8] => 0x3A5,         # \Upsilon, \Phi
     [9,0xA] => 0x3A8,       # \Psi, \Omega
+    0x18 => 0xB8,           # \c, cedilla
```
Comment (edemaine) [inline src/fonts/OTF/TeX/makeFF]: This line is duplicated in the same table below, and should be removed.

**Comment by ylemkimon** [comment]
Most fonts display D, G, K, L, N, R with cedilla, ḐḑĢģĶķĻļŅņŖŗ, with comma below, except for lower g which has a descender, hence comma above, for Latvian and Livonian. (https://github.com/KaTeX/katex-fonts/issues/2#issuecomment-362142563)

**Comment by edemaine** [comment]
@ylemkimon I think that character is not in standard LaTeX fonts, so should wait for another PR.

**Comment by ylemkimon** [comment]
@edemaine It seems so, except lower g:
image

**Comment by edemaine** [comment]
@ylemkimon Is that the LaTeX output of `\c{g}`? If so, we should see how `\c` is implemented in LaTeX...

**Comment by ylemkimon** [comment]
@edemaine Yes, it is.

**Comment by edemaine** [comment]
@ylemkimon You must be using an additional package.  Here's what I get with `pdflatex` on
```latex
\c{D} \c{d} \c{G} \c{g} \c{K} \c{k} \c{L} \c{l} \c{N} \c{n} \c{R} \c{r}
```
image

**Comment by ylemkimon** [comment]
@edemaine I was testing on quicklatex.com and sharelatex.com, so I'm not sure which packages they have installed, but here is the log:
```
This is pdfTeX, Version 3.14159265-2.6-1.40.18 (TeX Live 2017) (preloaded format=pdflatex 2017.7.12)  1 FEB 2018 15:14
entering extended mode
 \write18 enabled.
 %&-line parsing enabled.
**main.tex
(/compile/main.tex
LaTeX2e <2017-04-15>
Babel <3.12> and hyphenation patterns for 84 language(s) loaded.
(/usr/local/texlive/2017/texmf-dist/tex/latex/base/minimal.cls
Document Class: minimal 2001/05/25 Standard LaTeX minimal class
) (/compile/output.aux)
\openout1 = `output.aux'.

LaTeX Font Info:    Checking defaults for OML/cmm/m/it on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for T1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OT1/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMS/cmsy/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for OMX/cmex/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    Checking defaults for U/cmr/m/n on input line 3.
LaTeX Font Info:    ... okay on input line 3.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <7> on input line 5.
LaTeX Font Info:    External font `cmex10' loaded for size
(Font)              <5> on input line 5.
[1{/usr/local/texlive/2017/texmf-var/fonts/map/pdftex/updmap/pdftex.map}] (/com
pile/output.aux) ) 
Here is how much of TeX's memory you used:
 24 strings out of 492990
 491 string characters out of 6132621
 54148 words of memory out of 5000000
 3670 multiletter control sequences out of 15000+600000
 3640 words of font info for 14 fonts, out of 8000000 for 9000
 1141 hyphenation exceptions out of 8191
 19i,6n,14p,108b,92s stack positions out of 5000i,500n,10000p,200000b,80000s
</usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfon
ts/cm/cmr10.pfb></usr/local/texlive/2017/texmf-dist/fonts/type1/public/amsfonts
/cm/cmr5.pfb>
Output written on /compile/output.pdf (1 page, 17105 bytes).
PDF statistics:
 16 PDF objects out of 1000 (max. 8388607)
 10 compressed objects within 1 object stream
 0 named destinations out of 1000 (max. 500000)
 1 words of extra memory for PDF output out of 10000 (max. 10000000)
```

**Comment by kevinbarabash** [comment]
@ylemkimon on quicklatex.com if you click "Choose Options" you can edit the document preamble.  By default it's:
```
\usepackage{amsmath}
\usepackage{amsfonts}
\usepackage{amssymb}
```

**Comment by edemaine** [comment]
I tried removing those from the preamble, and still got the same results on quicklatex, which still differ from my local texlive installation (though maybe a different version?). Weird...

**Comment by ylemkimon** [comment]
@kevinbarabash Yes, but removing all three packages still gives the same result.

**Pull request merged**